styles/main.css

```diff
@@ -373,8 +373,9 @@ img {
 
 @media screen and (max-width: 520px) {
 
-    .bio table+thead,tfoot,
-    th {
+    .bio table thead,
+    .bio table tfoot,
+    .bio table th {
```

Comment:
In selector lists like this, each selector needs to be fully specified
The `+` combinator is a sibling combinator.  Did you mean `>`?